packages/web/src/lib/wallectconnect-mode.ts

```diff
@@ -57,7 +56,7 @@ export class WallectConnectSession {
 				try {
 					await this.close();
 				} catch (e) {
-					console.error("Can't close walletconnect connection", e);
+					console.log(e);
```

Comment:
why changing this?

Reply:
while testing, I must have changed it, will revert back.